src/content/Concerts/93-94Fall.md

```diff
@@ -0,0 +1,39 @@
+---
+# Concert Name (e.g. "Fall Concert")
+concertName: "Fall Concert"
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#F58025"
+# Date YY-MM-DD (for sorting)
+date: "1993-10-00"
+# Season
+season: "1993/94"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: "mozart"
+# Google Calendar link (if applicable)
+calendar: ""
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Little Russia
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = October 1993
+
+Section
+
+Overline = Program
+Work = Igor Stravinsky ~ *Symphonies of Wind Instruments*
+Work = Wolfgang Amadeus Mozart ~ Symphony No. 41 in C major, K. 551 *Jupiter*
+Break = --- Intermission ---
+Work = Pyotr Ilyich Tchaikovsky ~ *Symphony No. 2 in C minor*, op. 17
```

Comment:
no italics for generic name; need subtitle "Little Russian"

Reply:
In italics after op. 17?

Reply:
Symphony No. 2 in C minor, op. 17, *Little Russian*

Reply:
Also need comma before Jupiter, right?

Reply:
oki. Yeah I already did that

Reply:
whops that should be italics. Github changed what I wrote